src/zone.rs

```diff
@@ -105,6 +122,20 @@ impl Zone {
             result
         })
     }
+
+    pub fn contains(&self, other: &Zone) -> bool {
+        return match (&self.boundary, &other.boundary) {
+            (&Some(ref mpoly1), &Some(ref mpoly2)) => {
+                let m_self: GGeom = mpoly1.into();
+                let m_other: GGeom = mpoly2.into();
+
+                // In GEOS, "covers" is less strict than "contains".
+                // eg: a polygon does NOT "contain" its boundary, but "covers" it.
+                m_self.covers(&m_other)
```

Comment:
hum I would prefer this to be a bit more wrapped into the rust-geos lib, but I don't see how to do this, so useles someone has an idea, I think it's fine for the moment.

(and later we might want to cache those `GGeom`)